src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -31,3 +31,11 @@ databaseChangeLog:
       file: db/changelog/changes/15-adding-column-cart_item_id-in-shopping_carts-table.yaml
   - include:
       file: db/changelog/changes/16-adding-shopping_carts-for-existing-users.yaml
+  - include:
+      file: db/changelog/changes/17-create-order-item-table.yaml
+  - include:
+      file: db/changelog/changes/18-create-order-table.yaml
+  - include:
+      file: db/changelog/changes/19-adding-foreign-keys-to-order-item-table.yaml
+  #- include:
+  #   file: db/changelog/changes/20-change-column-in-order-table.yaml
```

Comment:
I committed this field because when creating the table, I accidentally set unique: tru. I fixed it in changeset 20 and the whole application works, but when calling mvn clean package the test kept failing